relay-server/src/metrics_extraction/event.rs

```diff
@@ -48,11 +48,11 @@ impl Extractable for Span {
 pub fn extract_metrics(
     event: &Event,
     spans_extracted: bool,
-    config: &MetricExtractionConfig,
+    config: CombinedMetricExtractionConfig,
     max_tag_value_size: usize,
     span_extraction_sample_rate: Option<f32>,
 ) -> Vec<Bucket> {
-    let mut metrics = generic::extract_metrics(event, config);
+    let mut metrics = generic::extract_metrics(event, config.clone());
```

Comment:
```suggestion
    let mut metrics = generic::extract_metrics(event, config);
```

`config` should be Copy

Reply:
I agree that is should be either a reference or `Copy`. Went for a reference now.